### ISSUE TITLE
Insiders build failure on 11/26/2019

### ISSUE THREAD

**Issue by ranasaria** [microsoft/azuredatastudio#8484]
The build that started yesterday night failed. https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=47713. I am investigating the failure. This is what the failure looks like:

https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=47716 :

<1s
Unhandled: Not found SourceFolder: e:\mssqltoolsagentworkspace\72\vsix
1s
<1s
Unhandled: Not found SourceFolder: e:\mssqltoolsagentworkspace\72\s\.build\win32-x64\user-setup\
1s
Cmd.exe exited with code '1'.

**Comment by kburtram** [comment]
@anthonydresser do you think this is related to recent pipeline definition changes?

**Comment by anthonydresser** [comment]
Yes this is because I removed the step that builds the vsixs from the package task as its own task, but this is the old release build.

If we wanted to keep it running, just need to add the ` yarn gulp package-external-extensions` step.

**Comment by kburtram** [comment]
@anthonydresser can we turn off the old builds once we no longer need them?  I don't think we'll need them once we've finished validating the new pipelines.

**Comment by ranasaria** [comment]
I think it is a good idea to run them in parallel for one week till we iron out issues like #8485

**Comment by anthonydresser** [comment]
Probably it makes sense to run them for the remainder of this week, but I don't think it is necessary past that if we don't find any serious issues.

**Comment by ranasaria** [comment]
Failure is understood, but we are turning off these old pipelines instead of fixing it.
